### ISSUE TITLE
Convert 'favorite' to AVU

### ISSUE THREAD

**Issue by michael-conway** [DICE-UNC/metalnx-web#30]
Currently Metalnx makes user favorites (stars) a database entry. This proposal is to make a favorite an AVU

* A favorite on a file or collection is a standard tag that includes the user who favorites the item
* The favorites database can be removed from irods-ext
* A conversion migration can be added to support this transition

This benefits by removing a source of drift and inaccuracy and makes favorites more generally useful among clients

**Comment by trel** [comment]
Need to define the specifics of A, V, and U.

Cloud browser currently uses...

```
A “Starred from Cloud Browser”
V <username>
U “iRODSUserTagging:Star”
```

Note the username does not use the “#Zone” notation, so potentially could create a namespace collision when used via Federation.  I’d propose including the “#Zone”.

**Comment by michael-conway** [comment]
Yes, MetaLnx should operate across federations. This will be eventually
very important so that virtual collections can be formed across federation
boundaries!
